src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -16,4 +17,12 @@
      * @throws CommandException If an error occurs during command execution.
      */
     public abstract CommandResult execute(Model model) throws CommandException;
+
+    public void setCommandInputString(String inputString) {
```

Comment:
Don't have much context here, this method doesn't seem necessary hahaah, not too sure

Reply:
It's actually necessary because I need to be able to store the command's input string (the actual text that the user types into the command box in the UI), otherwise I would not be able to allow the user to press up and down to navigate to previously typed commands.